config/rbac/role.yaml

```diff
@@ -45,6 +45,14 @@ rules:
   verbs:
   - delete
   - list
+- apiGroups:
+  - cdi.kubevirt.io
+  resources:
+  - datavolumes
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
oh, i guess this is needed here too, clusterkubevirtadm/cmd/credentials/credentials.go

Reply:
@davidvossel - done